### PR TITLE
Add single-use alipay bindings helpers

### PR DESCRIPTION
**Pull request by bdorfman-stripe**
Also add notes about sources which auto set currency to EUR

Fixes https://github.com/stripe/stripe-ios/issues/733

### CODE REVIEW

File: Stripe/PublicHeaders/STPSourceParams.h [hunk 1]
```diff
@@ -16,145 +16,151 @@ NS_ASSUME_NONNULL_BEGIN
...
- *  @return an STPSourceParams object populated with the provided values.
+ Creates params for a Bancontact source.
+ @see https://stripe.com/docs/bancontact#create-source
+ 
+ @param amount               The amount to charge the customer in EUR.
+ @param name                 The full name of the account holder.
+ @param returnURL            The URL the customer should be redirected to after they have successfully verified the payment.
+ @param statementDescriptor  (Optional) A custom statement descriptor for the payment.
+
+ @note The currency for Bancontact must be "eur". This will be set automatically for you.
```
Comment (bg-stripe) [inline Stripe/PublicHeaders/STPSourceParams.h]: 👍

File: Tests/Tests/STPSourceFunctionalTest.m [hunk 2]
```diff
@@ -295,6 +295,31 @@ - (void)testCreateSource_threeDSecure {
     [self waitForExpectationsWithTimeout:5.0f handler:nil];
 }
 
+- (void)testCreateSource_alipay {
+    STPSourceParams *params = [STPSourceParams alipayParamsWithAmount:1099
```
Comment (bg-stripe) [inline Tests/Tests/STPSourceFunctionalTest.m]: nit: colon alignment

**Comment by bdorfman-stripe** [comment]
Colon alignment fixed. I tested it out in test mode in our custom example (locally swapped out the sofort implementation) and it seems to work correctly.

File: Stripe/STPSourceParams+Private.h [hunk 3]
```diff
@@ -14,8 +14,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, nullable, strong) NSString *redirectMerchantName;
 
-//
-
```
Comment (joeydong-stripe) [inline Stripe/STPSourceParams+Private.h]: thx

**Comment by joeydong-stripe** [comment]
👍

**Pull request merged**